lib/MqttClient.js

```diff
@@ -404,9 +404,19 @@ class MqttClient {
                  *   "command": "go_to",
                  *   "spot_id": "Somewhere"
                  * }
+                 *
+                 * or (with xxx and yyy being coordinates)
+                 *
+                 * {
+                 *   "command": "go_to",
+                 *   "spot_id": [xxx, yyy]
+                 * }
                  */
                 case CUSTOM_COMMANDS.GO_TO:
-                    if (msg.spot_id) {
+                    // msg.spot_id is an array of exactly 2 values considered to be coordinates
```

Comment:
Try to rework the existing code, so there is only one this.vacuum.goTo call in there.